biome.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "https://biomejs.dev/schemas/1.6.4/schema.json",
+  "organizeImports": {
+    "enabled": true
+  },
+  "linter": {
+    "enabled": true,
+    "rules": {
+      "a11y": {
+        "useValidAriaProps": "warn",
+        "useValidAriaValues": "warn",
+        "noAriaUnsupportedElements": "warn",
+        "useAriaPropsForRole": "warn",
+        "useAltText": "warn",
+        "noBlankTarget": "off"
+      }
+    }
+  },
+  "formatter": {
+    "enabled": true,
+    "indentStyle": "space",
+    "indentWidth": 2,
+    "lineEnding": "lf",
+    "include": ["apps/website/src/", "apps/chatbot/src/"],
```

Comment:
I think the old prettier setup formatted everything in the repo that it knew how to handle, including anything at the root?